### ISSUE TITLE
Accessibility (trivial fix I think)

### ISSUE THREAD

**Issue by mfsjr** [mbrn/material-table#1375]
**Describe the bug**
Chrome browser extensions that check accessibility (like WAVE and aXe) report that the table's search field is missing a form label. 

Organizations in the U.S. have been forced to pay damages for inaccessible websites, so using material-table with accessibility errors could be a problem.

**To Reproduce**
1. Install WAVE extension on Chrome
2. Go to https://material-table.com/#/docs/features/search (or any page using search)
3. Run WAVE
4. See errors for "Missing form label", search field will be one of them

Expected behavior should be no error, there should be a visuallyhidden label "search" for that search field.

This is a great component, thanks for all your good work!

Seems like this could be a 1 minute fix for someone familiar with the codebase.

**Comment by aitchiss** [comment]
I've raised a PR that addresses this issue - hope it helps :)

**Comment by mfsjr** [comment]
Thank you @aitchiss !
